homeassistant/components/steam_online/sensor.py

```diff
@@ -107,9 +110,13 @@ def update(self):
             }.get(self._profile.status, STATE_OFFLINE)
             self._name = self._profile.persona
             self._avatar = self._profile.avatar_medium
+            self._last_online = self._get_last_online()
+            self._level = self._get_level()
         except self._steamod.api.HTTPTimeoutError as error:
             _LOGGER.warning(error)
-            self._game = self._state = self._name = self._avatar = None
+            self._game = (
+                self._state
+            ) = self._name = self._avatar = self._last_online = self._level = None
```

Comment:
See above.

Reply:
Changed it to single line initialization to None